crates/cast/src/helpers/scarb_utils.rs

```diff
@@ -69,18 +70,47 @@ pub fn get_scarb_manifest() -> Result<Utf8PathBuf> {
     Ok(path)
 }
 
-pub fn get_scarb_metadata(manifest_path: &Utf8PathBuf) -> Result<scarb_metadata::Metadata> {
+pub fn get_scarb_metadata(
+    manifest_path: &Utf8PathBuf,
+    with_deps: bool,
+) -> Result<scarb_metadata::Metadata> {
```

Comment:
Since we have only the `with_deps` flag for now, I think it's redundant to add the `ScarbOpts` struct. 
If the number of additional arguments increases, we may introduce `ScarbOpts`.

Reply:
I wouldn't say it's redundant - you would get `default()`, easier to understand syntax (no positional flags) and easier path to extend args.

Reply:
ok, makes sense
done